app/common/collections/types.py

```diff
@@ -176,6 +177,29 @@ def get_value_for_json_export(self) -> list[ChoiceDict]:
         return self.choices
 
 
+class DateAnswer(SubmissionAnswerBaseModel):
+    answer: date
+
+    @property
+    def _render_answer_template(self) -> str:
+        return "common/partials/answers/date.html"
+
+    def get_value_for_submission(self) -> dict[str, Any]:
+        return self.model_dump(mode="json")
+
+    def get_value_for_form(self) -> date:
+        return self.answer
+
+    def get_value_for_expression(self) -> date:
+        return self.answer
+
+    def get_value_for_text_export(self) -> str:
+        return self.answer.isoformat()
```

Comment:
Change to use `model.dump` for consistency - produces the same output of yyy-mm-dd

Reply:
Did this for the json export below, but doing this here doesn't work as we don't want json, we want a string eg "2023-04-05" so have left as-is for text_export